### PR TITLE
Add auto_update & auto_scroll flags

### PR DESCRIPTION
**Pull request by dxg**
* Added auto_update flag (`--auto-update 5000`)
* Added auto_scroll flag (`--auto-scroll`)

### CODE REVIEW

File: aha.c [hunk 1]
```diff
@@ -308,6 +322,31 @@ int main(int argc,char* args[])
...
+				auto_update
+			);
+		}
+		if (auto_scroll)
+		{
+			printf(
+				"<script type=\"text/javascript\">"
+				"  setInterval("
+				"   function () {"
+				"      console.log('here', document.body.scrollHeight);"
```
Comment (benkitzelman) [inline aha.c]: oops.. cleanup?

File: aha.c [hunk 2]
```diff
@@ -179,6 +181,18 @@ int main(int argc,char* args[])
 		else
 		if ((strcmp(args[p],"--stylesheet")==0) || (strcmp(args[p],"-s")==0))
 			stylesheet=1;
+		if (strcmp(args[p],"--auto-scroll")==0)
+			auto_scroll=1;
```
Comment (RenWenshan) [inline aha.c]: need `else` here

File: aha.c [hunk 3]
```diff
@@ -179,6 +181,18 @@ int main(int argc,char* args[])
 		else
 		if ((strcmp(args[p],"--stylesheet")==0) || (strcmp(args[p],"-s")==0))
 			stylesheet=1;
+		if (strcmp(args[p],"--auto-scroll")==0)
```
Comment (RenWenshan) [inline aha.c]: need else above this line

File: aha.c [hunk 4]
```diff
@@ -308,6 +324,27 @@ int main(int argc,char* args[])
...
+				"</script>\n",
+				auto_update
+			);
+		}
+		if (auto_scroll)
+		{
+			printf(
+				"<script type=\"text/javascript\">"
+				"window.addEventListener('load', function() {\n"
+        "  setTimeout(function() { window.scrollTo(0, document.body.scrollHeight); }, 50);\n"
```
Comment (dxg) [inline aha.c]: *cough*

**Pull request merged**